src/bootstrap/main.py

```diff
@@ -219,35 +213,42 @@ def init_reinit(directory, args):
 #
 
 
-def wrap(argv):
+def wrap():
     start = os.getcwd()
     try:
         topdir = find_west_topdir(start)
     except WestNotFound:
         sys.exit('Error: not a Zephyr directory (or any parent): {}\n'
                  'Use "west init" to install Zephyr here'.format(start))
-    # Put the top-level west source directory at the highest priority
-    # except for the script directory / current working directory.
-    sys.path.insert(1, os.path.join(topdir, WEST, 'src'))
-    main_module = importlib.import_module('west.main')
-    main_module.main(argv=argv)
+
+    # Replace the wrapper process with the "real" west
+
+    # sys.argv[1:] strips the argv[0] of the wrapper script itself
+    argv = [sys.executable,
+            os.path.join(topdir, WEST, 'west', 'src', 'west', 'main.py')] \
+           + sys.argv[1:]
```

Comment:
I was never a fan of that recommendation, so I ignore it. When you see the `(`, you don't know whether some intermediate result is being calculated (think `(a + b)/2`), or if someone just really hates `\`. The `\` makes things clearer.

Are we doing strict PEP 8?

Reply:
Guido hates `\` IIRC.

We had discussed coming up with a flake8 configuration file at some point in the past but never came up with a solution, so I thought the idea was just to keep things clean based on what it outputs at default settings; did I forget something?

Reply:
I'd say just skip the tool and use our common sense.

Reply:
I actually disagree. Keeping the flake8 output clean may be a bit of extra work, but it means that any time it complains, we can be sure it's based on just the changes introduced by the patchset we are working on. Which includes real bugs sometimes (e.g. flake8 complained that platform was undefined in this patchset when it ran as part of my editor's default config for opening a python file).

Reply:
I suspect `flake8` might be confused in this case. It has no problem with this code, for example:

```python3
def f(x):
    return 1 \
           + 2
```

In general, `flake8` doesn't seem to care whether you use `\` or brackets for line continuations. Probably because PEP 8 is a bit wishy-washy on it ("is preferred").

Looking [at the warning itself](https://lintlyci.github.io/Flake8Rules/rules/E127.html), it doesn't make any sense in this case. What it's saying is "hey, I think this code tries to visually align stuff, but it isn't correctly aligned". It's not saying "visual indent is bad". I misread it like that too.

I think it's a bad idea to change stuff just because some tool says so, unless you can think of a way in which it's bad. At that point, it doesn't matter if it was pointed out by a tool or not.

`flake8` and `pylint` (the more OCD one) and the like are nice helpers. Don't take them as dogma.

Breaking rules often makes sense.

Reply:
Would you be open to using `noqa: xxx` comments on each line that flake8 complains about?

It catches real bugs often enough that cluttering up its output seems like a mistake.

Reply:
Not a fan of cluttering up code with stuff like that in general. I'm more of a remove-everything guy. Might've noticed. :P

Would you be fine with just having `flake8` report a few warnings, or suppressing them in a configuration file?

Reply:
`flake8` found a single unused variable in Kconfiglib at least. That's still a "run occassionally to check for issues" to me.

I think most of that stuff can be avoided just by working hard to keep code minimalistic and boring, which is why I avoid clutter (except I probably over-comment stuff sometimes...).